Extensions/ShareTo/InitialViewController.swift

```diff
@@ -66,14 +66,13 @@ class InitialViewController: UIViewController, ShareControllerDelegate {
             if destinations.contains(ShareDestinationReadingList) {
                 profile.readingList?.createRecordWithURL(item.url, title: item.title ?? "", addedBy: UIDevice.current.name)
             }
-            
+
             if destinations.contains(ShareDestinationBookmarks) {
-                profile.bookmarks.shareItem(item)
+                profile.bookmarks.shareItem(item) >>> {
```

Comment:
Just to be on the safe side, I would use `.upon()` instead of `>>>` here. `.upon` will always be called and passes the closure a maybe type so you can check for errors. This way we'll always call this logic even if we decide to fail at a later point.